### PR TITLE
Preparation for AW-1

### PR DESCRIPTION
**Pull request by DavidMinarsch**
This PR:
- restructures navigation in docs
- adds content on incentivised test-nets in general and aw-1 in particular
- adds minimal docs on ledger v2

### CODE REVIEW

File: docs/css/my-styles.css [hunk 1]
```diff
@@ -438,17 +438,17 @@ html.debug nav ul li a {
     grid-column: 1 / span 3;
     grid-row: 1;
     border: solid 1px lightgrey;
```
Comment (DavidMinarsch) [inline docs/css/my-styles.css:440]: Temporary..needs proper styling

File: docs/i_nets/index.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Incentivized testnets. ...General intro to incentivized testnets;
```
Comment (DavidMinarsch) [inline docs/i_nets/index.md]: Placeholder text. @CobrasCobras please add

File: docs/i_nets/quickstart-aw1.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+
+## Intro
```
Comment (DavidMinarsch) [inline docs/i_nets/quickstart-aw1.md]: WIP, subject to change

File: docs/index.md [hunk 4]
```diff
@@ -1,23 +1,37 @@
-<div id="qst" class="grid-container">
+<div id="qst1" class="grid-container">
   <div class="grid-item item1">
     <h4>Welcome to Fetch.ai developer resources. Let's get started.</h4>
   </div>
+</div>
+<div id="qst" class="grid-container">
+  <div class="grid-item item1">
+    <h4>Another box goes here to draw attention to latest running testnet and prizes on offer, colour different!</h4>
```
Comment (DavidMinarsch) [inline docs/index.md]: @CobrasCobras headline needed

File: docs/ledger/index.md [hunk 5]
```diff
@@ -1,27 +1,7 @@
-# Fetch.ai ledger
```
Comment (DavidMinarsch) [inline docs/ledger/index.md:1]: still present, moved to `ledger_v1` dir

File: docs/ledger/index.md [hunk 6]
```diff
@@ -1,27 +1,7 @@
...
-
-The protocol uses a Proof-of-Stake mechanism to construct a blockchain, a Decentralised Random Beacon to elect a
-group of nodes that collaboratively build and decide upon the next block that will be entered into the ledger, and a Directed Acyclic Graph for coordination and notarization.
-
-For better understanding of our consensus protocol, please read [this blog post](https://medium.com/fetch-ai/a-minimal-agency-consensus-scheme-2a9a2fae7d26) and [the whitepaper](https://www.fetch.ai/uploads/Fetch.AI-A-Minimum-Agency-Consensus-Paper.pdf).
-
-## Support for on-chain ML and AI operations
-
-Coming soon.
+<div class="grid-item item_whole_row no-padding">
```
Comment (DavidMinarsch) [inline docs/ledger/index.md:2]: Main index site for ledger

File: docs/ledger_v1/index.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
+
+<div class="admonition note">
```
Comment (DavidMinarsch) [inline docs/ledger_v1/index.md:2]: Copy past from old `ledger` dir, added note on top

File: docs/ledger_v2/index.md [hunk 8]
```diff
@@ -0,0 +1,158 @@
+# Getting Ready for Mainnet V2
```
Comment (DavidMinarsch) [inline docs/ledger_v2/index.md:1]: @CobrasCobras text you provided me with

File: docs/playground.md [hunk 9]
```diff
@@ -1,61 +0,0 @@
-## The most basic introduction
```
Comment (DavidMinarsch) [inline docs/playground.md:1]: not needed anymore

File: theme/custom_theme/base.html [hunk 10]
```diff
@@ -78,7 +78,7 @@
                         <ul>
                             <li><a href="https://developer.fetch.ai/Bounty">Bounty</a></li>
                             <li><a href="https://explore.fetch.ai">Explorer</a></li>
-                            <li><a href="https://testnet-wallet.fetch.ai/">Testnet wallet</a></li>
+<!--                             <li><a href="https://testnet-wallet.fetch.ai/">Testnet wallet</a></li> -->
```
Comment (DavidMinarsch) [inline theme/custom_theme/base.html:81]: very problematic and confusing, removed

**Review by CobrasCobras — CHANGES_REQUESTED** [review]
Some initial comments:

- Some of the links under "Key links and information" on the "Getting Ready for Mainnet V2" page are not links, they are just plain text
- On home, change "Another box goes here to draw attention to latest running testnet and prizes on offer, colour different!" to read:

> ** Fetch.ai Incentivized testnets are running __now__**
> You can learn more about this journey at [link to the Introduction diagram]. The current running testnet is Agent themed, and you can get part of a substantial reward pool over the coming weeks. Each themed testnet divides its focus up into different pieces, so there's something for everyone. Get started now [link to agent world]!

@steevi may wish to review the "Getting Ready for Mainnet V2" piece given the links, end-points, etc. And we do need to point at the right inventivised testnet and not just agent land. 

Also, in "Getting Ready for Mainnet V2: there is a subsection that is labeled:
> Converting Agents to be ready for Mainnet v2
> Ask David M to fill this in and link to correct place in the AEA docs.

Replace "Incentivized testnets. ...General intro to incentivized testnets;" with:
> ## The Journey to mainnet version 2
> Fetch.ai is running three themed, incentivized testnets to take us to mainnet version 2. Each testnet will run for approximately 6 weeks, and will be arranged in groups of incentives. The below diagram outlines how this will work. When each incentivized testnet is complete, it will be reset and the new one created. This will reset state, and testing and work will focus on the new functionality added. With each step along the way, we get closer and closer to a fully featured Fetch V2 mainnet.
>
> Throughout this process, Fetch.ai is running a __stable testnet__ called Agentland. Agentland will not have its state reset, and is perfect for application and agent development and deployment.
>
> When the funky incentivized testnet is complete, mainnet v2 will be launched.

File: docs/ledger_v2/index.md [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+
+**Chain ID:** agent-land
+
+**Coin denomination:** atestfet
+
+**Decimals:** 18 (i.e. 1 testfet = 10^18 atestfet)
+
+**RPC Endpoint:** [https://rpc-agent-land.fetch.ai:443](https://rpc-agent-land.fetch.ai/)
+
+**Token Faucet:** https://faucet-agent-land.fetch.ai/claim
```
Comment (steevi) [inline docs/ledger_v2/index.md]: This is plain text, as it will not reply to GET requests. It will reply only on POST with parameters `Address=<address>`

**Pull request merged**